baybe/targets/base.py

```diff
@@ -37,17 +37,14 @@ def to_objective(self) -> SingleTargetObjective:
         return SingleTargetObjective(self)
 
     @abstractmethod
-    def transform(self, data: pd.DataFrame) -> pd.DataFrame:
-        """Transform data into computational representation.
-
-        The transformation depends on the target mode, e.g. minimization, maximization,
```

Comment:
is this not true anymore or do you consider this unnecessary?

Reply:
still true, but whenever I work on a function I re-read its docstring and somehow found that this one it is completely trivial, i.e. it actually reads like `the way the method performs its operation depends on the values of the attribues`, which is a trivial statement in the class context. But if you think I criticized too harshly, we can also keep it. Let me know what you prefer, and I can undo if necessary

Reply:
ok fine

Reply:
alright. Plus: the content I deleted does anyway not exist on the base class level. So we're basically referring to things that haven't even been defined at this point 🙃